Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM node:10-slim
-
-ARG npm_version="1.68.0"
```

Comment:
if npm_version is removed, then the following lines using it should be adjusted I guess?

Reply:
ARG is not necessary. Or is it a requirement for Actions?

Reply:
not really, but there's currently a LABEL line that uses the ARG value right below (probably results in an empty label I suppose?), and more concerning the install line for vsce would result in `npm install -g vsce@` (empty version), which is at best confusing (I guess it's equivalent to `npm install -g vsce@latest`?)

Initially my thinking was to keep this ARG to maintain some determinism in the action. But it assumed I was going to keep that ARG somewhat up-to-date, which I clearly have not :)
So I'm fine dropping the version label, and making the npm install explicitly `npm install -g vsce@latest`.

Reply:
I removed everything related to ARG node version. Also use `npm install -g vsce` which will install latest version without `@latest`.

To me I would delete all those labels. This is a simple image that is built once, launched, and then stopped and deleted. That data is not important from my perspective. But if you have those, you most probably have a reason for that.